docs/debug_e2etests.md

```diff
@@ -36,3 +39,63 @@ For example:
 ```bash
 kubectl get azureconfigs
 ```
+
+
+## Using Helm with the control plane cluster
+
+[The Custom Resources](https://github.com/giantswarm/apiextensions/tree/master/helm) required for the tests are installed as Helm charts, so it would be a good idea to install Helm and inspect the installed charts.
+
+There are several different Helm releases, and we need to use one that's compatible with the currently installed `tiller`.
+Let's check our `tiller` version:
+
+```bash
+kubectl get deployment -n kube-system -l name=tiller,app=helm -o jsonpath="{..image}"
+```
+
+We know the version that we need, let's install Helm:
+
+```bash
+export HELM_VERSION="v2.16.2"
+curl -L "https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz" > ./helm.tar.gz
+tar xzvf helm.tar.gz
+chmod u+x linux-amd64/helm
+sudo mv linux-amd64/helm /usr/local/bin/
+```
+
+Now we can list the installed charts
+
+```bash
+helm list
+```
+
+Or even get the values that were used to install those charts
+
+```bash
+helm get values <release-name>
+```
+
+
+## Inspecting resources on the cloud provider
+
+The operator will create resources on the cloud provider, like the tenant clusters. Sometimes it's useful to go to the provider console and see what's going on.
+On Azure we can find the resource groups being used to hold the tenant clusters.
```

Comment:
Missing here in which account they are - I know this should be obvious but there are multiple GS subscriptions.

Reply:
What would be best here, to specify the subscription id?

Reply:
Maybe something like `accountsgiantswarm` subscription? Not sure how cool it is to put the subscription id into a public repo, but not sure :)

Reply:
Added name of the subscription